seed/challenges/01-responsive-web-design/applied-visual-design.json

```diff
@@ -1143,7 +1143,7 @@
         }
       ],
       "description": [
-        "<code>Google Fonts</code> is a great way to find free fonts to use in a project.",
+        "<code><a href='https://fonts.google.com/' target='_blank'>Google Fonts</a></code> is a great way to find free fonts to use in a project.",
```

Comment:
I think we should add it now when we're at it :blush:

@freeCodeCamp/moderators What do you think about this link? How should it be formatted?